changelogs/client_server/newsfragments/1644.clarification

```diff
@@ -0,0 +1 @@
+Clarify specification by adding `/logout` to the overview list of endpoints that don't take a request body.
```

Comment:
Note that the other two in the list *do* take a request body; it's just not a JSON object. Could you reword slightly?